microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/TracingLogger.java

```diff
@@ -0,0 +1,42 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import static org.jboss.logging.Logger.Level.INFO;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "EMPTRAC", length = 4)
```

Comment:
What is the 'E' for? :)
Apart from that I parse this to something like MicroProfile Trace/-ing

Reply:
Eclipse, I believe. I based this on @jmesnil's MP-Config:

https://github.com/jmesnil/wildfly/blob/babea3b0f925fab044a3e9a04bfe30788f9b58c5/microprofile/config-smallrye/src/main/java/org/wildfly/extension/microprofile/config/smallrye/MicroProfileConfigLogger.java#L37

Reply:
@jmesnil Do you still want to follow the project code convention at https://github.com/wildfly/wildfly/pull/11320/files#diff-93ee82755e25dc17584f3efd8b7fde11R37 now that the code is being moved into WIldFly itself?

I'm ok with it (and what Juraci's done here) if you do; I just wanted to double check in case it was something you'd have changed if you'd noticed.

Reply:
@bstansberry thanks for noticing i, I forgot about this.

We should change that to WFLYTRAC (and I'll do the same for the mp-config subsystem).
I used the EMPCONF code before the subsystem was brought into WildFly codebase. There is no good reason to diverge from the existing convention of WFLYXXX.

@jpkrohling could you update it please?

Reply:
Done